assets/javascripts/discourse/templates/components/remind-assigns-frequency.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="controls controls-dropdown">
     <label>{{i18n "discourse_assign.reminders_frequency.description"}}</label>
     <ComboBox
-      @id="remind-assigns-frequency"
+      id="remind-assigns-frequency"
```

Comment:
the lack of `...attributes` applies to all attrs (yeah, select-kit is not glimmer ready 😉)